open-sphere-plugins/csv-common/src/main/java/io/opensphere/csvcommon/ui/controller/ImportableColumnController.java

```diff
@@ -46,6 +48,22 @@ public void detectNonImportableColumns(CSVParseParameters parameters, Preference
 
                 List<String> columnsToNotImport = CSVColumnPrefsUtil.getCustomKeys(registry, key);
 
+                List<?> currentParams = new ArrayList<String>();
+                currentParams = parameters.getColumnNames();
+                List<String> newParams = new ArrayList<String>();
+
+                Iterator<?> litr = currentParams.iterator();
+                while (litr.hasNext())

```

Comment:
Could replace this while iterator/while loop with currentParams.forEach()

Reply:
replaced